src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+        ball1.toString();
+        ball2.toString();
+        ball3.toString();
```

Comment:
1. let's use for loop to create and print random balls
2. calling toString() method on balls won't print info about them, let's use System.out.println